src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Guid.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Text;
+
+namespace System.Text.Json
+{
+    public ref partial struct Utf8JsonWriter
+    {
+        /// <summary>
+        /// Writes the <see cref="Guid"/> value (as a JSON string) as an element of a JSON array.
+        /// </summary>
+        /// <param name="value">The value to be written as a JSON string as an element of a JSON array.</param>
+        /// <exception cref="JsonWriterException">
+        /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
+        /// </exception>
+        public void WriteStringValue(Guid value)
+        {
+            if (_writerOptions.Indented)
+            {
+                if (!_writerOptions.SkipValidation)
+                {
+                    ValidateWritingValue();
+                }
+                WriteStringValueIndented(value);
+            }
+            else
+            {
+                if (!_writerOptions.SkipValidation)
+                {
+                    ValidateWritingValue();
+                }
+                WriteStringValueMinimized(value);
+            }
+
+            _currentDepth |= 1 << 31;
+            _tokenType = JsonTokenType.String;
+        }
+
+        private void WriteStringValueMinimized(Guid value)
+        {
+            // Calculated based on the following: ',"Guid value"'
+            int bytesNeeded = JsonConstants.MaximumGuidLength + 1;
```

Comment:
`MaximumGuidLength` seems to be for the part within the quotes.  Should this instead be `+ 3` to account for the open and close quote, too?

Reply:
Yep, will fix and add tests.